ogc/src/main/scala/geotrellis/server/ogc/OgcSourceRepository.scala

```diff
@@ -22,23 +22,23 @@ import io.circe.Json
 import higherkindness.droste.{Algebra, scheme}
 import jp.ne.opt.chronoscala.Imports._
 
-import java.time.ZonedDateTime
-
 case class OgcSourceRepository(store: List[OgcSource]) extends Repository[List] {
   def find(query: Query): List[OgcSource] = OgcSourceRepository.eval(query)(store)
 }
 
 object OgcSourceRepository {
   import geotrellis.store.query.QueryF._
 
-  def algebgra: Algebra[QueryF, List[OgcSource] => List[OgcSource]] = Algebra {
+  def algebra: Algebra[QueryF, List[OgcSource] => List[OgcSource]] = Algebra {
     case Nothing()           => _ => Nil
     case All()               => identity
     case WithName(name)      => _.filter(_.name == name)
     case WithNames(names)    => _.filter(rs => names.contains(rs.name))
-    case At(t, fn)           => _.filter(_.metadata.attributes.get(fn).map(ZonedDateTime.parse).fold(false)(_ == t))
-    case Between(t1, t2, fn) => _.filter {
-      _.metadata.attributes.get(fn).map(ZonedDateTime.parse).fold(false) { current => t1 >= current && t2 < current }
+    case At(t, _)           => _.filter(_.time.getOrElse(null) == t)
```

Comment:
`null`?

Reply:
The idea here was to succinctly return false if our OGCSource Option[ZonedDateTime] is None and it looks like `zonedDateTime == null` returns false. Is there a better way to do this comparison? We want None == t to be false and otherwise compare on the two date times.

Reply:
@CloudNiner ` _.filter(_.time.exists(t))`